### ISSUE TITLE
rocketqav2 损失函数

### ISSUE THREAD

**Issue by fangxiang00** [Changanyue/bert-downstream-framework#3]
你好：
    torch版RocketQA v2的复现和论文有出入吧？一个是dual encoder的logits计算，另一个是损失函数，论文里是KL + cross loss.
    你这里的改写有验证比较过吗

**Comment by Changanyue** [comment]
是有出入的，还没有验证过
